### PR TITLE
bcm2835 : deploy pkgconfig files as well

### PR DESCRIPTION
**Pull request by LongChair**
In our distro, mpv will now fetch a Rpi environment checking pkgconfig files. so this commit adds them to sysroot.

### CODE REVIEW

**Comment by MilhouseVH** [comment]
Aren't those pkgconfig files representative of the Raspbian environment rather than LibreELEC?

eg. mmal.pc:

```
prefix=/opt/vc
exec_prefix=${prefix}
libdir=${exec_prefix}/lib
includedir=${prefix}/include

Name: MMAL
Description: Multi-Media Abstraction Layer library for RPi
Version: 1
Requires: vcsm
Libs: -L${libdir} -lmmal -lmmal_core -lmmal_util -lmmal_vc_client -lbcm_host
Cflags: -I${includedir}
```

Although I don't have any real objection, approving.

File: packages/graphics/bcm2835-driver/package.mk [hunk 1]
```diff
@@ -45,6 +45,9 @@ make_target() {
     ln -sf $SYSROOT_PREFIX/usr/lib/libbrcmEGL.so $SYSROOT_PREFIX/usr/lib/libEGL.so
     ln -sf $SYSROOT_PREFIX/usr/lib/libbrcmGLESv2.so $SYSROOT_PREFIX/usr/lib/libGLESv2.so
     cp -PRv $FLOAT/opt/vc/lib/*.a $SYSROOT_PREFIX/usr/lib
+
+  mkdir -p $SYSROOT_PREFIX/usr/lib/pkgconfig
+    cp -PRv $FLOAT/opt/vc/lib/pkgconfig/* $SYSROOT_PREFIX/usr/lib/pkgconfig
```
Comment (MilhouseVH) [inline packages/graphics/bcm2835-driver/package.mk]:
```
cp -PRv $FLOAT/opt/vc/lib/pkgconfig $SYSROOT_PREFIX/usr/lib
```
and you wouldn't need the `mkdir`
?

Comment (lrusak) [inline packages/graphics/bcm2835-driver/package.mk]: @LongChair can you update it?

Comment (LongChair) [inline packages/graphics/bcm2835-driver/package.mk]: @lrusak @MilhouseVH : yeah will update it no problem, just have been away from home for work. I'll update this PR on monday :)

**Comment by LongChair** [comment]
Updated :)

**Pull request merged**